mmdet3d/visualization/local_visualizer.py

```diff
@@ -598,21 +595,16 @@ def _draw_pts_sem_seg(self,
                           points: Union[Tensor, np.ndarray],
                           pts_seg: PointData,
                           palette: Optional[List[tuple]] = None,
-                          ignore_index: Optional[int] = None):
+                          ignore_index: Optional[int] = None) -> None:
```

Comment:
Please make sure that the visualization of the `lidar_seg` task can run successfully.

Reply:
Have checked.